<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_device_tablet</class>
 <widget class="QDialog" name="Edit_device_tablet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="OK_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>690</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background: rgb(45, 81, 135);
border-radius: 12px;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>331</width>
     <height>450</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Battery:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>camera spec:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="os_lineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="price_lineEdit"/>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="des_lineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>OS:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="processor_lineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Processor:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Resolution:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="battery_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="quantity_lineEdit"/>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="resolution_lineEdit_2"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="display_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="brand_lineEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="resolution_lineEdit_1"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Quantity:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>STorage:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Display:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="storage_lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Brand:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="model_lineEdit"/>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="camera_spec_lineEdit"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Bluetooth:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="bluetooth_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>481</width>
     <height>372</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Weight:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="brightness_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Dimension:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="speaker_lineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Brightness:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Internal Memory:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="dimension3_lineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="dimension2_lineEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="water_proof_with_controller">
      <property name="text">
       <string>Water Proof</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Speaker:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QCheckBox" name="iris_with_controller">
      <property name="text">
       <string>Iris</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>External Memory:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="wifi_lineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Wifi:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="internal_Memory"/>
    </item>
    <item row="7" column="3">
     <widget class="QCheckBox" name="face_with_controller">
      <property name="text">
       <string>Face</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="have_pen_with_controller">
      <property name="text">
       <string>Have pen</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="dimension1_lineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="external_memory_lineEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="weight_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="finger_with_controller">
      <property name="text">
       <string>Fingerprint</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
color: rgb(0, 85, 127);</string>
   </property>
   <property name="text">
    <string>EDIT (TABLET)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
